clients/samples/example_hipblaslt_preference.cpp

```diff
@@ -64,9 +64,9 @@
 #define DIM2 1024
 #define DIM3 1024
 #define BATCH_COUNT 1
-#define ALPHA 2
-#define BETA 3
-#define BENCH_LOOP_COUNT 3
+#define ALPHA 1
+#define BETA 0
+#define BENCH_LOOP_COUNT 20
```

Comment:
if you want to increase the loop count, it better to add extra argument to control it.